prov/verbs/src/fi_verbs.c

```diff
@@ -938,29 +954,49 @@ static void fi_ibv_msg_ep_qp_init_attr(struct fi_ibv_msg_ep *ep,
 	}
 }
 
+static inline struct fi_info *fi_ibv_find_info(const char *fabric_name,
+		const char *domain_name)
+{
+	struct fi_info *info;
+	int match = 0;
+
+	for (info = verbs_info; info; info = info->next) {
+		if ((domain_name && !strcmp(info->domain_attr->name, domain_name))
+			|| (fabric_name && !strcmp(info->fabric_attr->name, fabric_name))) {
+			match = 1;
+			break;
```

Comment:
just return info here, no need for match variable, and return NULL below
nit: move the '||' to the end of the first line, rather than the start of the next